### DISCUSSION TITLE
请问抢bw周边购买资格有脚本吗

### DISCUSSION THREAD

**Discussion by kaseeen** [mikumifa/biliTickerBuy#398] [提问专区]
![Screenshot_2024-07-09-15-33-36-16_149003a2d400f6adb210d7e357a3a646](https://github.com/mikumifa/biliTickerBuy/assets/115888450/02917031-bde2-4a5c-8104-6cfd5c7b3ad9)

**Comment by dimensioncity** [comment]
目前无解
